src/main/java/com/getbase/jenkins/plugins/metrics/history/influxdb/generators/JenkinsBasePointGenerator.java

```diff
@@ -61,9 +89,10 @@ public JenkinsBasePointGenerator(Run<?, ?> build) {
                 .addField(BUILD_DURATION, duration)
                 .addField(QUEUING_DURATION, action.getQueuingDurationMillis())
                 .addField(TOTAL_DURATION, duration + action.getQueuingDurationMillis())
-                .addField(BUILD_STATUS_MESSAGE, build.getBuildStatusSummary().message);
+                .addField(BUILD_STATUS_MESSAGE, build.getBuildStatusSummary().message)
+                .fields(fields);
```

Comment:
What if `fields` are null? Wouldn't it break the data point sending later on? Or Influx just accepts that?

Reply:
`fields = null` will not break the pipeline. Points are written to influx db correctly. Checked with `Backup` pipeline